### PR TITLE
Added visual feedback flash messages when sending mail from the mail form

### PR DESCRIPTION
**Pull request by Xiaoming94**
Added visual feedback flash messages when sending mail from the mail form

### CODE REVIEW

File: app/controllers/contact_controller.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def create
         GroupMailer.anonymous_mail(mail_array, contact_params[:body], contact_params[:title]).deliver_now
       end
 
-     redirect_to :contact_index
+      redirect_to :contact_index, flash: {notice: "your mail has been sent to " + contact_params[:to_whom].first}
```
Comment (Kalior) [inline app/controllers/contact_controller.rb]: Shouldn't this show all of the contacted organs, not just the first one?

Comment (Xiaoming94) [inline app/controllers/contact_controller.rb]: The array `contact_params[:to_whom]` contains just the email address of the reciever (i.e. Styrit@chalmers.it when emailing styrit) and two empty strings. You can try to print it out yourself with `p contact_params[:to_whom].to_s` to check you self :)

Comment (Kalior) [inline app/controllers/contact_controller.rb]: Yeah sure, but you should (I believe) also be able to email several committees, all of those should show up, don't you think?

Comment (Xiaoming94) [inline app/controllers/contact_controller.rb]: Lol, Just realised those were checkboxes and not radio buttons, my bad. Fixing it right now

**Comment by Xiaoming94** [comment]
@Vilddjur Could you take a look at this? 😄

**Comment by Vilddjur** [comment]
Looks good!

File: app/controllers/contact_controller.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ def create
         GroupMailer.anonymous_mail(mail_array, contact_params[:body], contact_params[:title]).deliver_now
       end
 
-     redirect_to :contact_index
+      redirect_to :contact_index, flash: {notice: "your mail has been sent to " + build_contacts_string(contact_params[:to_whom])}
```
Comment (Kalior) [inline app/controllers/contact_controller.rb]: A final comment: the text should be localized so that we can have an English version and a Swedish version.

**Comment by Xiaoming94** [comment]
@Kalior Think I have completed the localisation now, mind having another look? 🙂

**Pull request merged**